js/packages/openinference-instrumentation-langchain/package.json

```diff
@@ -12,12 +12,19 @@
     "url": "https://github.com/Arize-ai/openinference.git"
   },
   "scripts": {
-    "prebuild": "rimraf dist & pnpm run version:update",
-    "build": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "prebuild": "rimraf dist && pnpm run version:update",
+    "build": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json && tsc-alias -p tsconfig.esm.json",
+    "postbuild": "echo '{\"type\": \"module\"}' > ./dist/esm/package.json && rimraf dist/test",
```

Comment:
same here - feels a tad weird that we build the tests. I guess that already was there though 😓